Crashing on Windows 10 for iOS

![image](https://user-images.githubusercontent.com/1362505/33682971-910172b8-dac9-11e7-8a75-a5a575d42412.png)

And Git tells that not a single modification has been done

Having almost the same error than before.

Its is not saying ```[RNV] undefined``` anymore now.

```
$ npm version major
v2.0.0

> RandomProject@1.0.0 postversion C:\Users\AdminUser\RandomProject
> react-native-version --never-amend

[RNV] Versioning Android...
[RNV] Android updated
[RNV] Versioning iOS...

  Usage: cli [options] [projectPath]

  Version your React Native app in a `npm version` fashion.

  Options:

    -V, --version                output the version number
    -a, --amend                  Amend the previous commit. Also updates the latest Git tag to point to the amended commit. This is done automatically when react-native-version is run from the "version" or "postversion" npm script. Use "--never-amend" if you never want to amend.
    --skip-tag                   For use with "--amend", if you don't want to update Git tags. Use this option if you have git-tag-version set to false in your npm config or you use "--no-git-tag-version" during npm-version.
    -A, --never-amend            Never amend the previous commit.
    -b, --increment-build        Only increment build number.
    -B, --never-increment-build  Never increment build number.
    -d, --android [path]         Path to your "android/app/build.gradle" file. (default: android/app/build.gradle)
    -i, --ios [path]             Path to your "ios/" folder. (default: ios)
    -L, --legacy                 Version iOS using agvtool (macOS only). Requires Xcode Command Line Tools.
    -q, --quiet                  Be quiet, only report errors.
    -r, --reset-build            Reset build number back to "1" (iOS only). Unlike Android's "versionCode", iOS doesn't require you to bump the "CFBundleVersion", as long as "CFBundleShortVersionString" changes. To make it consistent across platforms, react-native-version bumps both by default. You can use this option if you prefer to keep the build number value at "1" after every version change. If you then need to push another build under the same version, you can use "-bt ios" to increment.
    -s, --set-build <number>     Set a build number. WARNING: Watch out when setting high values. This option follows Android's app versioning specifics - the value has to be an integer and cannot be greater than 2100000000. You cannot decrement this value after publishing to Google Play! More info at: https://developer.android.com/studio/publish/versioning.html#appversioning
    -t, --target <platforms>     Only version specified platforms, eg. "--target android,ios".
    -h, --help                   output usage information
[RNV] Done, with errors.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! RandomProject@1.0.0 postversion: `react-native-version --never-amend`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the RandomProject@1.0.0 postversion script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\AdminUser\AppData\Roaming\npm-cache\_logs\2018-04-23T15_03_26_167Z-debug.log
```

--- a/index.js
+++ b/index.js
@@ -396,7 +396,10 @@ function version(program, projectPath) {
 					}, [])
 					.forEach(function(err) {
 						if (program.outputHelp) {
-							err.text && log(err);
+							log(
+								Object.assign({ style: "red", text: err.toString() }, err),
+								programOpts.quiet
+							);
 						}
 					});
 

